cura/Settings/MachineManager.py

```diff
@@ -550,6 +550,18 @@ def copyValueToExtruders(self, key: str):
             if extruder_stack != self._active_container_stack and extruder_stack.getProperty(key, "value") != new_value:
                 extruder_stack.userChanges.setProperty(key, "value", new_value)  # TODO: nested property access, should be improved
 
+    ## Copy the value of all settings of the current extruder to all other extruders as well as the global container.
+    @pyqtSlot()
+    def copyAllValuesToExtruders(self):
+        for key in self._active_container_stack.userChanges.getAllKeys():
+            new_value = self._active_container_stack.getProperty(key, "value")
+            extruder_stacks = [stack for stack in ExtruderManager.getInstance().getMachineExtruders(self._global_container_stack.getId())]
```

Comment:
It's better to use the global stack to get all extruders, so we don't depend on the `ExtruderManager`, which is something we want to remove later. You can do it like this:
```
extruder_stacks = list(self._global_container_stack.extruders.values())
```